app/controllers/upvs_controller.rb

```diff
@@ -48,6 +53,11 @@ def fetch_callback_url(registered_urls)
     raise CallbackError, :malformed_callback
   end
 
+  def fetch_profile(assertion)
```

Comment:
nestrcime toto rovno do repository/service ?

Reply:
ano, vytiahnem to von